atomix/cluster/src/test/java/io/atomix/cluster/impl/DefaultClusterMembershipServiceTest.java

```diff
@@ -86,6 +86,8 @@ void shouldGetLocalMember() {
     assertThat(membershipService.getLocalMember())
         .as("the local member is member 0")
         .isEqualTo(expectedMember);
+
+    membershipService.stop().join();
```

Comment:
:wrench: Would like to have such a thing in the after method, but I guess this is because some tests are closing this pro-actively.

Reply:
Yeah exactly, most tests start and stop the membership service ad-hoc.